R/reactives.R

```diff
@@ -441,7 +441,8 @@ reactive <- function(x, env = parent.frame(), quoted = FALSE, label = NULL,
   # Attach a label and a reference to the original user source for debugging
   if (is.null(label))
     label <- sprintf('reactive(%s)', paste(deparse(body(fun)), collapse='\n'))
-  srcref <- attr(substitute(x), "srcref")
+  srcref <- attr(substitute(x), "srcref", exact = TRUE)
+  label <- srcrefToLabel(srcref[[1]], label)
```

Comment:
This will override the label if one is supplied with `reactive(label = "foo")`. Is that the intent?